### PR TITLE
Initial version of pip-reqs

### PR DESCRIPTION
**Pull request by GaretJax**

### CODE REVIEW

File: pip-reqs/setup.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    @staticmethod
+    def longdesc():
+        return Setup.read('README.rst') + '\n\n' + Setup.read('CHANGELOG.rst')
+
+    @staticmethod
+    def shortdesc():
+        node = ast.parse(Setup.read(os.path.join(PACKAGE, '__init__.py')))
+        docstring = ast.get_docstring(node)
+        return docstring.split('\n\n')[0].strip().replace('\n', ' ')
```
Comment (stefanfoulis) [inline pip-reqs/setup.py]: huh? whats all this? Seems a bit excessive ;-)

**Comment by stefanfoulis** [comment]
👍

File: pip-reqs/setup.py (hunk 1 above)
Comment (GaretJax) [inline pip-reqs/setup.py]: Just copy paste from my boilerplate :D

Comment (GaretJax) [inline pip-reqs/setup.py]: (allows you to import stuff in the __init__.py file and not have it executed during installation)

**Pull request merged**